_drafts/SC1-2-1+SC1-2-3-video-transcript.md

```diff
@@ -0,0 +1,113 @@
+---
+name: video element transcript
+group: SC1-2-video-element
+description: |
+  Non-streaming video elements must have all audio and visual information available in a transcript.
+
+success_criterion:
+- 1.2.1 # Audio-only and Video-only (Prerecorded)
+- 1.2.3 # Audio Description or Media Alternative (Prerecorded)
+- 1.2.8 # Media Alternative (Prerecorded)
+
+test_aspects:
+- DOM Tree
+- CSS Styling
+- Audio output
+- Visual output
+
+authors:
+- Wilco Fiers
+- Brian Bors
+---
+
+## Test Procedure
+
+### Applicability
+
+The rule applies to any [non-streaming][] video element [visible in the page][]
+
+### Expectation 1
+
+A [text transcript][] is available for the test target, either on the page or available through a link (or some other mechanism).
+
+### Expectation 2
+
+The [text transcript][] (from expectation 1) contains all the visual and auditory information of the test target.
```

Comment:
Should we define 'transcript'? In other words, there is nothing in here that checks to verify that the transcript is not, for example, a scanned image of text.

Reply:
Transcript links to a definition. That definition does not exclude scanned images of text, but such images would fail a different SC and should pass this one.